terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/welcome/index.jsp

```diff
@@ -60,6 +60,10 @@
 			<td><a id="EL" href="${pageContext.request.contextPath}/el">EL
 					Function Test</a></td>
 		</tr>
+        <tr>

```

Comment:
@ikeyat 
 Sorry, It's difference that may occur due to I had to rebase too much.
 I Canceled the wrong rebase, and re-push only the necessary commits.
 This difference does not occur.